code/online_challenges/src/hackerrank/Electronics_Shop/Electronics_Shop.java

```diff
@@ -0,0 +1,83 @@
+// Part of Cosmos by OpenGenus
+import java.io.*;
+import java.math.*;
+import java.text.*;
+import java.util.*;
+import java.util.regex.*;
+
+public class Solution {
```

Comment:
Reformat using 4 spaces.

Reply:
I changed it to 4 spaces but it automatically changes to 8 spaces.

Reply:
@Riyagoel2 What are you using? Try the IntelliJ code formatter.

Reply:
I have made the changes .